.github/labeler.yml

```diff
@@ -6,7 +6,13 @@
 
 "Category: Build System":
 - build/*
+- win32/build/*
 - scripts/**/*
+- '**/*.m4'
+- '**/*.w32'
+- buildconf.bat
+- buildconf
+- configure.ac
```

Comment:
Please keep the entries sorted.

Reply:
Like the current commit? in my world * comes after the z, but some may disagree

Reply:
This is how VSCode sorts it:

```yaml
"Category: Build System":
- '**/*.m4'
- '**/*.w32'
- build/**/*
- buildconf
- buildconf.bat
- configure.ac
- scripts/**/*
- win32/build/**/*
```

To me that makes the most sense, but I'm not care too much.

Reply:
here ya go, VS Code compliant sort